man/trollolo.1.md

```diff
@@ -83,16 +97,124 @@ conventions for Scrum boards, and reports burndown data.
 
 ### get-cards -- Get card data for a board
 
+`trollolo get-cards --board-id=<boad id>`
+
 Read all card data for a given board.
 
 ### get-checklists -- Get checklist data for a board
 
+`trollolo get-checklists --board-id=<boad id>`
+
 Read all checklist data for a given board
 
+### get-description -- Get description of a card
+
+`trollolo get-description --card-id=<card id>`
+
+Read description of a given card.
+
 ### get-lists -- Get list data for a board
 
+`trollolo get-lists --board-id=<boad id>`
+
 Read all list data for a given board.
 
+### get-raw -- Get raw JSON from Trello API
+
+`trollolo get-raw <url fragment>`
+
+Read raw JSON from Trello using the given URL fragment. Trollolo adds the server
+part and API version as well as the credentials from the Trollolo
+configuration.
+
+### backup -- Create local copy of a board
+
+`trollolo backup --board-id=<board id>`
+
+Save a local copy of a board as a JSON file. The backup will be stored in 
+'~/.trollolo/backup/<board-id>/'.
+
+### list-backups -- List all backups
+
+`trollolo list-backups`
+
+Get a list of all local backups.
+
+### show-backup -- Show local backup
+
+`trollolo show-backup --board-id=<board id>`
+
+Show the local backup of the given board.
+
+### list-member-boards -- List name and id of boards for a user
+
+`trollolo list-member-boards --member-id=<member id>`
+
+Get a list of all boards the given user is a member of.
+
+### set-cover -- Set picture as cover
+
+`trollolo set-cover <filename> --card-id=<card id>`
+
+Use the given file as the cover for the given card.
+
+### make-cover -- Make existing picture the cover
+
+`trollolo make-cover <filename> --card-id=<card id>`
+
+Make the given picture the cover for the given card. The given card must
+be an attachment of the given card. If you want to use a new picture use
+`set-cover`.
+
+### organization -- Get organization details
+
+`trollolo organization --org-name=<organization name>`
+
+Get details of an organization.
+
+### organization-members -- Get a list of organization member 
+
+`trollolo organization-members --org-name=<organization name>`
+
+Get a list of all organization members.
+
+### set-description -- Write description to a card
+
+`trollolo set-description --card-id=<card id>`
+
+Write description to the given card. The description is read from STDIN, use
+<^D> to end input.
+
+### setup-scrum -- Setup Scrum boards
+
+`trollolo setup-scrum`
+
+Create boards, lists and labels with names configured in '~/.trollolorc' or
+with the defaults.
+
+### move-backlog -- Move backlog from planning to sprint board
+
+`trollolo move-backlog --planning-board-id=<planning board id>
+--sprint-board-id=<sprint board id>`
+
+Move cards from the 'Backlog' list on the planning board to the 'Sprint
+Backlog' list on the sprint board.
```

Comment:
What about mentioning here that this is what is doing at the beginning of the sprint once the planning meeting is finished? :thinking:

Reply:
I think anyone that uses this tool to manage their Scrum should know when to move the cards from the planning board to the sprint board / the other way around but I can add a sentence for each (`move-backlog` and `cleanup-sprint`)

Reply:
yes, but it is difficult to read. If we add that it is used in this case it is much clear.